LINUX.md

```diff
@@ -801,6 +801,23 @@ Run `docker run hello-world`, you should see something like:
 
 
 
+## TLDR
+
+Add TLDR - a modern addition to MAN pages, which will help you find nice documentation and examples on most Linux commands:
+
+```bash
+echo "alias tldr='docker run --rm -it -v ~/.tldr/:/root/.tldr/ nutellinoit/tldr'" >> ~/.aliases
+```
+
+You can try `tldr` with:
```

Comment:
Top, je rajouterais deux trucs
1. "It's normal that it takes ~1 minute the first time, as the cache needs to be built. Subsequent calls will be fast."
2. Est-ce qu'il faut faire un `source ~/.aliases` juste après le `echo`, que les alias soient pris en compte ?